hedera-node/configuration/testnet/upgrade/feeSchedules.json

```diff
@@ -1378,7 +1378,93 @@
                 }
             } ]
         }
-    }, {
+    },
+        {
```

Comment:
Every entry is repeated twice in this file. Need to do the same for this entry.

Reply:
Done. I've added it. Great catch.